src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqTransportBindingElement.cs

```diff
@@ -65,23 +70,48 @@ public override string Scheme
         /// <summary>
         /// The largest receivable encoded message
         /// </summary>
-        public override long MaxReceivedMessageSize { get; set; }
+        public override long MaxReceivedMessageSize
+        {
+            get
+            {
+                return _maxReceivedMessageSize;
+            }
+
+            set
+            {
+                if (value < 0 || value >= MaxRabbitMqMessageSize)
+                {
```

Comment:
This will throw if the value is exactly MaxRabbitMqMessageSize. I think this was supposed to be `>` and not `>=`. The zero check should also be `<=` as it doesn't make sense to set a value of 0.

Reply:
I just looked at the code for MaxReceivedMessageSize on .NET Framework and when the value \<= 0, it throws `new ArgumentOutOfRangeException(nameof(value), value,
SR.ValueMustBePositive)`. You can keep the single throw if you want, but the constructor overload you are using would be better to be switched to this one as you name the argument and provide the value too. You also missed adding any text to the resource string file, you left a place holder.